tensorflow/python/grappler/auto_mixed_precision_test.py

```diff
@@ -263,12 +264,19 @@ def setUp(self):
     # to ignore performance and always transform the graph.
     self._original_ignore_perf_value = os.getenv(self.IGNORE_PERF_VAR)
     os.environ[self.IGNORE_PERF_VAR] = '1'
+    # This test does not run with XLA: disable it
+    self._original_tf_xla_flags = os.getenv(self.TF_XLA_FLAGS)
```

Comment:
I think you can use the `test_util.disable_xla` annotation to disable it. [See here for an example.](https://github.com/tensorflow/tensorflow/blob/754ac36f54db34d303a60eb08d34199a7945e576/tensorflow/python/kernel_tests/array_ops_test.py#L1128).

For the description, you could put something like "This test does not pass with XLA".